src/main/java/core/basesyntax/IDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface IDraw {
```

Comment:
Interface names shouldn't be verbs. I'd recommend renaming it to Drawable or something like this